docs/tools/wifi.sh

```diff
@@ -28,21 +33,40 @@ case "$os" in
 	(Linux)
 		echo "Detected Linux"
 		echo "Re-mounting the EFI partition"
-		sudo umount /dev/nvme0n1p1 || echo
-		sudo mkdir /tmp/apple-wifi-efi || echo
-		sudo mount /dev/nvme0n1p1 /tmp/apple-wifi-efi || echo
+		if [[ ${1-default} = -v ]]
+		then
+			sudo umount /dev/nvme0n1p1 || true
```

Comment:
`mount`, and `umount` could have `-v` for the verbose mode

Reply:
a0aeb83